modules/KoreBuild.Tasks/UpgradeExternalDependencies.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using KoreBuild.Tasks.Utilities;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Framework;
+using System.Collections.Generic;
```

Comment:
Nit:  we typically [sort System usings](http://dailydotnettips.com/2015/01/06/automatically-place-the-system-directives-first-when-sorting-usings-option-in-visual-studio/) before others.